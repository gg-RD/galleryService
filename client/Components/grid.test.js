import React from 'react';
import { shallow, mount, render } from 'enzyme';
import Grid from './grid'
import sinon from 'sinon'
import styled from 'styled-components';
import renderer from 'react-test-renderer'

export const Img = styled.img `
	  border-style: none;
	  padding: 5px;
	  height: 350px;
	  width: 250px;
`


describe('Grid Component', () => {
  it('renders without crashing', () => {
    const wrapper = shallow(<Grid />);
  });

  it()
});