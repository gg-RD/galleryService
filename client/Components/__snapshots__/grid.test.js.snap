// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it works 1`] = `
<img
  className="sc-bwzfXH eyZrOz"
/>
`;
